CMakeLists.txt

```diff
@@ -92,6 +78,7 @@ include(XsdFu)
 include(GTest)
 include(Doxygen)
 include(HeaderTest)
+find_package(OME 5.2.0 REQUIRED Compat Common)
```

Comment:
Do we need an hardcoded version number here or can we default to `find_package(OME REQUIRED Compat Common)` for now ?